README.md

```diff
@@ -0,0 +1,21 @@
+# census
+
+Serverless deployment steps:
+<ul>  
```

Comment:
you don't actually have to use html for readme styling! you can use markdown since this readme is a .md file. 
here's markdown for lists for example: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists